src/sql/postgres/protocol/ErrorResponse.zig

```diff
@@ -106,6 +398,17 @@ pub fn toJS(this: ErrorResponse, globalObject: *jsc.JSGlobalObject) JSValue {
         }
     }
 
+    // Parse key/value from detail for unique constraint violations
```

Comment:
are there other error codes we can attach info for? this is just for unique constraints currently